Themes/default/languages/Profile.english.php

```diff
@@ -283,9 +283,7 @@
 
 // Use numeric entities in the below three strings.
 $txt['no_reminder_email'] = 'Unable to send reminder email.';
-$txt['send_email'] = 'Send an email to';
-$txt['to_ask_password'] = 'to ask for your authentication details';
-
+$txt['send_email_to_webmaster'] = 'Send an email to <a href="mailto:%1$s">webmaster</a> to ask for your authentication details.';
```

Comment:
I believe the current directive is to avoid adding HTML to language files, ref: #4479

Reply:
Ok :)